
$FreeBSD$
  $MCom: ports/graphics/gimp-devel/files/patch-gimptool-2.0.in,v 1.3 2005/06/28 05:40:04 adamw Exp $

--- gimptool-2.0.in.orig	Tue Jan 15 12:21:05 2002
+++ gimptool-2.0.in	Fri May 17 20:46:18 2002
@@ -215,19 +215,19 @@
     | --uninstall-script | --uninstall-admin-script )
       case $1 in
         --*install-bin)
-          install_cmd="@INSTALL_PROGRAM@"
+          install_cmd="@INSTALL_SCRIPT@"
           install_dir="$HOME/@gimpdir@/plug-ins"
           ;;
         --install-bin-strip)
-          install_cmd="@INSTALL_PROGRAM@ -s"
+          install_cmd="@INSTALL_SCRIPT@ -s"
           install_dir="$HOME/@gimpdir@/plug-ins"
           ;;
         --*install-admin-bin)
-          install_cmd="@INSTALL_PROGRAM@"
+          install_cmd="@INSTALL_SCRIPT@"
           install_dir="$gimpplugindir/plug-ins"
           ;;
         --install-admin-bin-strip)
-          install_cmd="@INSTALL_PROGRAM@ -s"
+          install_cmd="@INSTALL_SCRIPT@ -s"
           install_dir="$gimpplugindir/plug-ins"
           ;;
         --*install-script)
